src/media-popup.js

```diff
@@ -54,4 +71,4 @@ ipcRenderer.on('load-media', (event, mediaUrl, mediaType) => {
     video.autoplay = true;
     video.playbackRate = 1;
   }
-});
+});
```

Comment:
While blank line isn't recommended, a newline character should be put at the end of the file.
(`});` is bad, `});\n` is good and `});\n\n` is bad too.)